ts/DataGrid/DataGridOptions.d.ts

```diff
@@ -259,6 +270,17 @@ export interface DataGridCellEvents {
      * Callback function to be called when the cell is no longer hovered.
      */
     mouseOut?: DataGridCellEventCallback;
+
+    /**
+     * Callback function to be called after editing of cell value.
+     */
+    afterEdit?: DataGridCellEventCallback;
+
+    /**
+     * Callback function to be called when the column is sorted
+     * (for instance, after clicking on header).
+     */
+    afterSorting?: DataGridColumnEventCallback;
```

Comment:
I believe this should be called `DataGridCellEventCallback` right?

Reply:
Nope, because "this"  refers to column, not cell.